Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml

```diff
@@ -25,12 +28,21 @@ query: |
   | where Active == true
   | where isnotempty(FileName)
   // using innerunique to keep perf fast and result set low, we only need one match to indicate potential malicious activity that needs to be investigated
-  | join kind=innerunique (
-    SecurityEvent 
+  | join kind=innerunique ( union isfuzzy=false 
+    (SecurityEvent 
     | where TimeGenerated between(starttime..endtime)
     | where EventID in ("4688","8002","4648","4673")
```

Comment:
This raises an issue with a change I asked you do in the past, and might have been a mistake: namely adding 4688 to detection rules that where about process. I think we should revert those.

Reply:
@oshezaf I added 4688 event filtering only to detections that conditioned on CommandLine not by process

Reply:
Great. So we were smarter than I remembered :-)